nodeclient/apimodels/core.go

```diff
@@ -315,21 +315,28 @@ type (
 		ValidatorStake iotago.BaseToken `serix:"3,mapKey=validatorStake"`
 		// FixedCost is the fixed cost that the validator receives from the total pool reward.
 		FixedCost iotago.Mana `serix:"4,mapKey=fixedCost"`
+		// Active indicates whether the validator was active recently, and would be considered during committee selection.
+		Active bool `serix:"5,mapKey=active"`
 		// LatestSupportedProtocolVersion is the latest supported protocol version of the validator.
-		LatestSupportedProtocolVersion iotago.Version `serix:"5,mapKey=latestSupportedProtocolVersion"`
+		LatestSupportedProtocolVersion iotago.Version `serix:"6,mapKey=latestSupportedProtocolVersion"`
 	}
 
 	// AccountStakingListResponse defines the response for the staking REST API call.
 	AccountStakingListResponse struct {
-		Stakers []*ValidatorResponse `serix:"0,mapKey=stakers"`
+		Stakers     []*ValidatorResponse `serix:"0,mapKey=stakers"`
+		PageSize    uint32               `serix:"1,mapKey=pageSize"`
+		Coursor     uint32               `serix:"2,mapKey=coursor"`
```

Comment:
```suggestion
		Cursor      uint32               `serix:"2,mapKey=cursor"`
```

Reply:
Tbh, I think the cursor needs to be a *string with omitempty... at least that's how it's done in the inx-indexer. :thinking:

Reply:
yeah, but you need there string to specify the query, and I simply use index, so I thought int is enough